<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InitWidget</class>
 <widget class="QWidget" name="InitWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lbHabitants">
       <property name="text">
        <string>Habitants</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="sbHabitants"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbSites">
       <property name="text">
        <string>Sites</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="sbSites"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbBornesSite">
       <property name="text">
        <string>Bornes/site</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="sbBornesSite"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lbVelos">
       <property name="text">
        <string>VÃ©los</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="sbVelos"/>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="btValider">
       <property name="text">
        <string>Valider</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
